src/dialogs/ShuffleDialog.java

```diff
@@ -65,6 +72,8 @@ of this software and associated documentation files (the "Software"), to deal
     private ArrayManager ArrayManager;
     private JFrame Frame;
     private UtilFrame UtilFrame;
+    List<Distributions> distributions;
+    List<String> distributionNames;
```

Comment:
why isn't visibility specified? what uses do these have as properties instead of local variables?

Reply:
They're used in more than one method. The methods it's used in are unrelated. That's the whole point of properties, is it not?

Reply:
oh oops, the former is, but I don't see the latter anywhere

Reply:
It was originally :)
I didn't realize it still wasn't.